ResourceManager/GitExtensionsControlInitialiser.cs

```diff
@@ -15,7 +15,11 @@ internal sealed class GitExtensionsControlInitialiser
 
         public GitExtensionsControlInitialiser(GitExtensionsFormBase form)
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
+            if (form.GetType().Name != "BugReportForm")
```

Comment:
Checking for a type by name like this is a code smell. What is wrong with passing a boolean around to control this?